src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      action="/"
+      method="get"
+      data-qa="big">
+      <label class="lable">
+        <img
+          class="img__form"
+          src="images/Search.svg"
+          alt="search__icon"
+         >
+
+        <input
+          class="input"
```

Comment:
Combine class names to combine styles in the CSS. Follow the checklist, please
```suggestion
          class="input input--big"
```